css/style.css

```diff
@@ -1,19 +1,189 @@
+:root{
+    --grey-dark: #363434;
+    --grey: #adadad;
+    --white: #fff;
+    --orange: #f05a28;
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
+}
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    box-sizing: border-box;
+    font-weight: 400;
+}
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 3rem;
+    border-bottom: 0.1rem solid hsl(0, 0%, 75%);
+}
+main {
+    display: flex;
+    flex-direction: column;
+    margin-top: -60px;
+ }
+
+ .logo{
+    display: block;
+    min-width: 1.5rem;
+    min-height: 1.5
+    rem;
+ }
+
+
+.list {
+    display: flex;
+    justify-content: center;
+    padding: none;
+    align-items: center;
+    flex-wrap: wrap;
+    list-style: none;
+
+ }
+
+.items{
+    display: flex;
+    padding: 0 1.5rem;
+} 
+
+.links{
+color: var(--grey);
+text-decoration: none;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.links:hover{
```

Comment:
Great 👍
This will change colour for mouse users who hover.
Which pseudo-class could you add to also change the colour for a keyboard user who has tabbed onto the link?